dfmanager/manager.go

```diff
@@ -30,6 +30,12 @@ func NewManager(prjKey []byte, prjName, fName string) (*Manager, error) {
 	return &Manager{srv: service, prj: prjName, fName: fName}, nil
 }
 
+// SetFilename sets provided name as filename
+func (m *Manager) SetFilename(name string) *Manager {
```

Comment:
actually, fields of the object are supposed to be immutable. Why do we need setter here?